docker/lambda.env

```diff
@@ -3,8 +3,7 @@
 
 # submitter env vars
 SUBMITTER_ENV_NAME=test
-SUBMITTER_THROTTLE=50
-SUBMITTER_DESTINATION_URL=http://antenna:8000/submit
```

Comment:
This updates the lambda environment we use for tests. This ditches the old variables so they go to default values and adds the new variable.